11/windows/windowsservercore-1809/Dockerfile

```
@@ -1,82 +1,82 @@
+
+ARG GIT_VERSION=2.26.0
+ARG GIT_PATCH_VERSION=1
+RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; `
+    $url = $('https://github.com/git-for-windows/git/releases/download/v{0}.windows.{1}/MinGit-{0}-64-bit.zip' -f $env:GIT_VERSION, $env:GIT_PATCH_VERSION) ; `
+    Write-Host "Retrieving $url..." ; `
+    Invoke-WebRequest $url -OutFile 'mingit.zip' -UseBasicParsing ; `
+    Expand-Archive mingit.zip -DestinationPath c:\mingit ; `
+    Remove-Item mingit.zip -Force
+
+ARG GIT_LFS_VERSION=2.10.0
+RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; `
+    $url = $('https://github.com/git-lfs/git-lfs/releases/download/v{0}/git-lfs-windows-amd64-v{0}.zip' -f $env:GIT_LFS_VERSION) ; `
+    Write-Host "Retrieving $url..." ; `
+    Invoke-WebRequest $url -OutFile 'GitLfs.zip' -UseBasicParsing ; `
+    Expand-Archive GitLfs.zip -DestinationPath c:\mingit\mingw64\bin ; `
+    Remove-Item GitLfs.zip -Force ; `
+    & C:\mingit\cmd\git.exe lfs install ; `
+    $CurrentPath = (Get-Itemproperty -path 'hklm:\system\currentcontrolset\control\session manager\environment' -Name Path).Path ; `
+    $NewPath = $CurrentPath + ';C:\mingit\cmd' ; `
+    Set-ItemProperty -path 'hklm:\system\currentcontrolset\control\session manager\environment' -Name Path -Value $NewPath
+
+ARG user=jenkins
+
+ARG AGENT_FILENAME=agent.jar
+ARG AGENT_HASH_FILENAME=$AGENT_FILENAME.sha1
+
+RUN net user "$env:user" /add /expire:never /passwordreq:no ; `
+    net localgroup Administrators /add $env:user ; `
+    Set-LocalUser -Name $env:user -PasswordNeverExpires 1; `
+    New-Item -ItemType Directory -Path C:/ProgramData/Jenkins | Out-Null
+
+ARG AGENT_ROOT=C:/Users/$user
+ARG AGENT_WORKDIR=${AGENT_ROOT}/Work
+
+ENV AGENT_WORKDIR=${AGENT_WORKDIR}
+
+# Get the Agent from the Jenkins Artifacts Repository
+RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; `
+    Invoke-WebRequest $('https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/{0}/remoting-{0}.jar' -f $env:VERSION) -OutFile $(Join-Path C:/ProgramData/Jenkins $env:AGENT_FILENAME) -UseBasicParsing ; `
+    Invoke-WebRequest $('https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/{0}/remoting-{0}.jar.sha1' -f $env:VERSION) -OutFile (Join-Path C:/ProgramData/Jenkins $env:AGENT_HASH_FILENAME) -UseBasicParsing ; `
+    if ((Get-FileHash (Join-Path C:/ProgramData/Jenkins $env:AGENT_FILENAME) -Algorithm SHA1).Hash -ne (Get-Content (Join-Path C:/ProgramData/Jenkins $env:AGENT_HASH_FILENAME))) {exit 1} ; `
+    Remove-Item -Force (Join-Path C:/ProgramData/Jenkins $env:AGENT_HASH_FILENAME)
+
+USER $user
+
+RUN New-Item -Type Directory $('{0}/.jenkins' -f $env:AGENT_ROOT) | Out-Null ; `
+    New-Item -Type Directory $env:AGENT_WORKDIR | Out-Null
+
+VOLUME ${AGENT_ROOT}/.jenkins
+VOLUME ${AGENT_WORKDIR}
+WORKDIR ${AGENT_ROOT}
```

Add newline at the end of the file
```suggestion
WORKDIR ${AGENT_ROOT}

```